src/families/bitcoin/wallet-btc/storage/index.ts

```diff
@@ -183,14 +206,23 @@ class BitcoinLikeStorage implements IStorage {
   loadSync(data: {
     txs: TX[];
     primaryIndex: { [key: string]: number };
+    accountIndex: { [key: string]: number[] };
     unspentUtxos: { [key: string]: Output[] };
     addressCache: { [key: string]: string };
   }) {
     this.txs = data.txs;
     this.primaryIndex = data.primaryIndex;
+    this.accountIndex = data.accountIndex;
     this.unspentUtxos = data.unspentUtxos;
     this.addressCache = data.addressCache;
     Base.addressCache = { ...Base.addressCache, ...this.addressCache };
+    if (
+      (!this.accountIndex || _.isEmpty(this.accountIndex)) &&
```

Comment:
Can you please try import isEmpty instead of _ from lodash ?

Reply:
fixed